site/docs/red-team/configuration.md

```diff
@@ -67,6 +67,58 @@ There are three core concepts that affect the number of generated redteam tests:
 - `hijacking`
   ...
 
+#### Custom Policies
+
+In addition to the predefined plugins, you can create one or more custom policies to test specific requirements or constraints of your application. Custom policies allow you to generate targeted redteam tests that are tailored to your business needs.
+
+##### Configuring Custom Policies
+
+To use a custom policy, add a `policy` field to your redteam configuration:
+
+```yaml
+redteam:
+  plugins:
+    - id: 'policy'
+      numTests: 10
+      config:
+        policy: 'Your custom policy statement here'
+```
+
+##### Examples of Custom Policies
+
+Here are some business examples of custom policies:
+
+1. **Financial Services**:
+
+   ```yaml
+   policy: 'Do not provide specific investment advice or recommend individual stocks.'
+   ```
+
+2. **Healthcare**:
+
+   ```yaml
+   policy: 'Do not diagnose medical conditions or prescribe treatments. Always refer users to qualified healthcare professionals.'
+   ```
+
+3. **E-commerce**:
+
+   ```yaml
+   policy: 'Do not disclose customer personal information or order details. Refer users to their account page for such information.'
+   ```
+
+4. **Education**:
+
+   ```yaml
+   policy: 'Do not write essays or complete assignments for students. Provide guidance and explanations instead.'
+   ```
+
+##### Best Practices for Custom Policies
+
+1. Be specific and clear in your policy statement.
+2. Align the policy with your application's purpose and ethical guidelines.
+3. Consider potential edge cases and loopholes when formulating your policy.
+4. Regularly review and update your custom policies as your application evolves.
```

Comment:
I think these could be improved to actually provide useful advice

- Be specific and clear 
- Enumerate edge cases and loopholes 
- When possible, write policies as affirmations rather than negations (e.g. "Prefer guidance and explanations" vs "Don't just give the answer" - this is [best practice](https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-the-openai-api#h_1f4c9c5fa1))

Let's update the example accordingly.  Just a single well-thought example is probably ok if that's easier.

We don't need to tell the dev to align with their guidelines, or to update when their stuff changes